RealmSwift/Realm.swift

```diff
@@ -1356,6 +1356,35 @@ extension Realm {
         return Unchecked(ret)
     }
 
+    /**
```

Comment:
wouldn't this be for swift less than 6, I'm a little bit confused because of the commit message

Reply:
This is moving the old version of the function into the `#if compiler(<6)` block.